### PR TITLE
added Apache Tomcat

### PR DESCRIPTION
**Pull request by Pallavi024**
NOTE:
- Please fill out the template below 

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Y

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Y

## Have you written this article by yourself? Have you taken inspiration from anywhere and then written it in your words?

> [Yes I have taken inspiration from youtube and wikipedia]

### CODE REVIEW

File: guides/Web Containers/Apache Tomcat.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Apache Tomcat
+
+Apache Tomcat, often referred to as Tomcat Server,is a Java servlet container and a web server from the Apache Software Foundation(ASF).
+It can be used standalone or it can be used behind traditional web servers such as Apache httpd.It uses the request/response message exchange pattern to serve web pages.
+Tomcat basically implements the java servlet and the java server pages specifications from Sun Microsystems.It provides a pure java web server enviornment for java code to run in.
+
+Apache Tomcat includes tools for configuaration and management.It can also be configured directly by editing xml configuration files.It is a cross platform software which works on various operating systems.Tomcat requires java version 1.6 although previous versions have java 1.1 through 1.5.
```
Comment (RN0311) [inline guides/Web Containers/Apache Tomcat.md]: 1) Spell check of "configuration".

Comment (RN0311) [reply guides/Web Containers/Apache Tomcat.md]: 2. Can you add something more to this article, if possible!

**Comment by the-ethan-hunt** [comment]
56% plagiarised

**Comment by RN0311** [comment]
@Pallavi024 : Try to lower down your plagiarism percent. Good luck :smile:

**Comment by the-ethan-hunt** [comment]
Merging this

**Pull request merged**